contracts/interfaces/0.8.x/IFilteredMinterV2.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+// Created By: Art Blocks Inc.
+
+import "./IFilteredMinterV1.sol";
+
+pragma solidity ^0.8.0;
+
+/**
+ * @title This interface extends the IFilteredMinterV1 interface in order to
+ * add support for manually setting project max invocations.
+ * @author Art Blocks Inc.
+ */
+interface IFilteredMinterV2 is IFilteredMinterV1 {
+    /**
+     * @notice Local max invocations for project `_projectId`, tied to core contract `_coreContractAddress`,
+     * updated to `_maxInvocations`.
+     */
+    event ManuallySetProjectMaxInvocations(
+        uint256 indexed _projectId,
+        address indexed _coreContractAddress,
+        uint256 _maxInvocations
+    );
+
+    // Sets the local max invocations for a given project, checking that the provided max invocations is
+    // less than or equal to the global max invocations for the project set on the core contract.
```

Comment:
I think we should add something along the lines saying, "This does not affect the value defined on the core contract, and only limits mints occurring on this minter".

Reply:
agreed. i think the inline comments above the actual implementation of the functions do a good job of describing this, so will probably just copy over most of that content to the interface inline docs